src/ontology/Makefile

```diff
@@ -4,7 +4,7 @@ ONT=foodon
 BASE=$(OBO)/$(ONT)
 SRC=$(ONT)-edit.ofn
 RELEASEDIR=../..
-ROBOT= robot
+ROBOT= robot --catalog catalog-v001.xml
```

Comment:
How this here ever worked? I dont know :D

Reply:
Robot wasn't being used directly in build process, it was only being involked manually on a file by file basis now and then.  We have been using ontofox fetches to build import files!

Reply:
What about running the release pipeline?